proxy/public/500.html

```diff
@@ -0,0 +1,2 @@
+<h1>500 Web server unavaiable 😵</h1>
+<p>Please contact datagovhelp@gsa.gov for assistance.</p>
```

Comment:
On existing FCS config, We have custom static page for 500 error hosted on filestore.data.gov. We can reuse it. Maybe a separated ticket after the clould.gov move.